plugins/woocommerce/client/blocks/assets/js/blocks/product-description/edit.tsx

```
@@ -50,7 +50,7 @@ function Content( props ) {
 	) : (
 		<div
 			{ ...blockProps }
-			dangerouslySetInnerHTML={ { __html: content?.rendered } }
+			dangerouslySetInnerHTML={ { __html: rawContent } }
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)